js/richobjectstringparser.js

```diff
@@ -0,0 +1,81 @@
+/* global OCA, Handlebars */
+
+/**
+ * @copyright (c) 2016 Joas Schilling <coding@schilljs.com>
+ *
+ * @author Joas Schilling <coding@schilljs.com>
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ */
+
+(function(OCA, Handlebars) {
+
+	OCA.SpreedMe.RichObjectStringParser = {
+
+		_userLocalTemplate: '<span class="mention-user">@{{id}}</span>',
```

Comment:
We should use name here, so we display the user displayname, not the uid

Reply:
I think it is better to show **@nickvergesen** than **@Joas Schilling** or **Joas Schilling**, as the @ would make easier to spot mentions (I guess that in the future there could be other bold texts), and the @ prepended to the display name feels strange. Not a strong opinion on this, though; @nextcloud/designers ?

Reply:
But `@some sha for ldap users` is not usable...

Reply:
It rather is `@597ae2f6-16a6-1027-98f4-d28b5365dc14` 😉

Reply:
Since names are bold, I think it should be fine with an `@`.
I checked WhatsApp and Threema are showing you the "real"name, Telegram shows the id.
However just see the LDAP thing above and I guess everyone knows we shouldn't use IDs ;)

Reply:
>  Since names are bold, I think it should be fine with an @.

Maybe instead of just using bold text we could use [a coloured background like in Riot](https://github.com/nextcloud/server/pull/7514#issuecomment-356304123). But, for the time being, I think that just bold is fine ;-)

>  However just see the LDAP thing above and I guess everyone knows we shouldn't use IDs ;)

Agreed ;-) I will change it to use the real name.